packages/react-tabster/src/TabsterContext.tsx

```diff
@@ -8,26 +8,22 @@ import { Types as AHTypes } from 'ability-helpers';
  * Context value, all members here can be undefined since the API relies heavily on window/document which are not
  * present during server render in SSR
  */
-export interface FocusManagementContextValue {
+export interface TabsterContextValue {
   /**
    * Ability helpers focusable API
    */
-  focusable?: AHTypes.FocusableAPI;
+  focusable?: Types.FocusableAPI;
 
   /**
    * Raw Ability helpers instance
    */
-  ahInstance?: AHTypes.AbilityHelpersCore;
+  tabsterInstance?: Types.TabsterCore;
 }
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-export const internal__FocusManagementContext = React.createContext<FocusManagementContextValue>(
-  ({} as unknown) as FocusManagementContextValue,
-);
+export const TabsterContext = React.createContext<TabsterContextValue>({} as TabsterContextValue);
```

Comment:
Will be this available publicly? I.e `import { TabsterContext } from ''`?

Reply:
IMO it shoulld be public. We're assuming too much when we enclosure it behind a hook and a provider. WDYT @layershifter ?

Reply:
If we are exporting context we are opening Pandora's box:

```tsx
function App() {
  return (
    <>
      <TabsterProvider /> // ✅ it's a public API that we want to share
      <TabsterContext.Provider /> // 🔴 we don't want to expose and feature this usage
    </>
  );
}
```

```tsx
useArrowNavigationGroup(); ✅ it's a public API that we want to share
useTabsterContext(); // 😕 not sure that we want to expose it
```